src/configuration/formatting-evaluator-config.ts

```diff
@@ -15,10 +15,10 @@ const regexType = Type.Record(Type.String(), Type.Number(), { minProperties: 1 }
 const htmlType = Type.Record(Type.String(), Type.Number(), {
   default: {
     br: 0,
-    code: 1,
+    code: 5,
```

Comment:
Is there another config to subtract words inside for this and block quote?

Reply:
You mean that you would count the element but not its content?

Reply:
Yes

Reply:
First easy implementation that comes to mind is this:
![image](https://github.com/user-attachments/assets/6e6e9662-5cba-4946-ae8c-d3cdc93b7198)

If that's okay I'll push the changes here.

Reply:
Notes after discussion: will rename to `countWords` and default to `false`.

Reply:
default to `true`